pkg/operator/starter.go

```diff
@@ -46,16 +42,6 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
-	if err != nil {
-		return err
-	}
-
-	v1helpers.EnsureOperatorConfigExists(
```

Comment:
@damemi as I review this again after seeing the e2e error, I'm thinking removing the EnsureOperatorConfigExists()  without doing something else to get the yaml processed isn't going to work.  If this call is removed, what's going to apply or create this yaml?  The CVO has no awareness of this content under bindata (ie its not within /manifests).

/hold

Reply:
@jboyd01 it is under manifests, part of this is moving that out of bindata to there https://github.com/openshift/cluster-svcat-apiserver-operator/pull/47/files#diff-72907faadd83d28353576644ccbb29c2

Reply:
Thanks, I see it now